stdlib/LinearAlgebra/test/diagonal.jl

```diff
@@ -458,6 +458,29 @@ end
     end
 end
 
+@testset "testing istriu, istril, isdiag for Diagonal" begin
+    D1 = Diagonal([[1 0; 3 4], [5 0; 7 8]])
+    D2 = Diagonal([[1 2; 3 4], [5 6; 7 8]])
+    D3 = Diagonal([[1 2; 0 4], [5 6; 0 8]])
+    D4 = Diagonal([[1 0; 0 4], [5 0; 0 8]])
+
+    @test istril(D1) == true
```

Comment:
Rather than `== true`, this should just be `@test istril(D1)`. Similarly, `== false` should be `@test !istriu(D1)`.